raft-tech/konfirm
# Feature/helm chart

# What's New?
This introduces a basic Helm chart for Konfirm along with some supporting Make commands.  
This PR also adds a new workflow for publishing the Helm chart to a Helm repo.
Needed to update manager's ClusterRole to be able to get, list, and watch k8s secrets.



# What's related?
Closes #12 

# How to test
1. pull this branch
2. run `make helm-install-kind` 
A kind cluster should be created for you with CRD's and Konfirm installed. 

# Notes
Helm chart releaser action reference: https://helm.sh/docs/howto/chart_releaser_action/

IMPORTANT: the helm chart release github workflow WILL NOT WORK until a secret called `GITHUB_TOKEN` is added to the repository containing credentials authorized to publish to Github Pages (what is used as the Helm chart repository)
